resharper/resharper-unity/src/Rider/UnityEditorProtocol.cs

```diff
@@ -138,6 +143,11 @@ private void CreateProtocols(FileSystemPath protocolInstancePath)
 
             myLogger.Info($"EditorPlugin protocol port {protocolInstance.Port} for Solution: {protocolInstance.SolutionName}.");
 
+            if (protocolInstance.ProtocolGuid != ProtocolCompatibility.ProtocolGuid)
+            {
+                OnOutOfSync(myComponentLifetime);
```

Comment:
Return after showing notification? Otherwise protocol will be create for incompatible rd versions

Reply:
thanks!